platform/database/tests/database_test.go

```diff
@@ -467,38 +469,47 @@ func TestProdDatabase(t *testing.T) {
 				name: "when a condition is provided it only gets the requested data",
 				condition: &struct {
 					sql string
-					arg []string
-				}{sql: "WHERE id IN (?)", arg: []string{firstData.ID}},
+					arg []interface{}
+				}{sql: "WHERE id IN (?)", arg: []interface{}{firstData.ID}},
 				shouldFindData: []testData{firstData},
 				shouldErr:      false,
 			},
 			{
 				name: "when multiple conditions are given it gets all the request data",
 				condition: &struct {
 					sql string
-					arg []string
-				}{sql: "WHERE id IN (?)", arg: []string{firstData.ID, secondData.ID}},
+					arg []interface{}
+				}{sql: "WHERE id IN (?)", arg: []interface{}{[]string{firstData.ID, secondData.ID}}},
```

Comment:
This might seem weird when you see it:

what `sqlx.In()` expects is a list of `interface{}` arguments and for each `(?)` it finds in the query, it will look at the corresponding element in this `[]interface{}` slice and if the element is a slice it will properly replace the (?) with the necessary number of arguments `$1, $2` in this case.